extensions/amp-analytics/0.1/vendors.js

```diff
@@ -63,6 +68,25 @@ export const ANALYTICS_CONFIG = {
           '${baseSuffix}'
     },
     'optout': '_gaUserPrefs.ioo'
+  },
+
+  'parsely': {
+    'requests': {
+      'host': 'https://srv.pixel.parsely.com',
+      'basePrefix': '{{host}}/plogger/?rand={{timestamp}}&idsite={{apikey}}&' +
```

Comment:
- I think you need to replace {{variable}} with ${variable}.
- amp-analytics.md in parent directory should be edited to have an entry for yourself. There, feel free to add all the required vars. If you would prefer, you can link to a page on your help docs and mention the details there.